posts/2020-11-09-ocp-log-forwarding.adoc

```diff
@@ -15,20 +15,20 @@ additional_authors:
 
-This blog post walks you through configuring the Log Forwarding API on your OCP cluster to forward your logs to a Fluentd and Splunk deployment that is external to your OCP cluster. The Fluentd server redirects the forwarded logs to Splunk by using Splunk's HTTP Event Collector (HEC) API. We also cover the configuration and deployment of the Fluentd and Splunk server on an external machine. The final steps integrate Open Liberty logs with the Log Forwarder and setup Splunk dashboards that are provided by the Open Liberty team.
+This blog post walks you through configuring the Log Forwarding API on your OpenShift cluster to forward your logs to a Fluentd and Splunk deployment that is external to your cluster.  We also cover the configuration and deployment of the Fluentd and Splunk server on an external machine. The Fluentd server redirects the forwarded logs to Splunk by using Splunk's HTTP Event Collector (HEC) API. In the final sections, you integrate Open Liberty logs with the Fluentd forwarder and setup Splunk dashboards that are provided by the Open Liberty team.
 
-If you would rather set up the Log Fowarding API, Fluentd forwarder, and Splunk locally within your OCP cluster, see link:https://www.openshift.com/blog/forwarding-logs-to-splunk-using-the-openshift-log-forwarding-api[Forwarding Logs to Splunk Using the OpenShift Log Forwarding API] on the OpenShift blog. After you complete those steps, proceed to the <<integrating-open-liberty-json-logs,Integrating Open Liberty JSON Logs>> and <<setting-up-splunk-dashboard,Setting Up Splunk-Dashboard>> sections of this blog post to complete your setup.
+If you would rather set up the Log Forwarding API, Fluentd forwarder, and Splunk locally within your OpenShift cluster, see link:https://www.openshift.com/blog/forwarding-logs-to-splunk-using-the-openshift-log-forwarding-api[Forwarding Logs to Splunk Using the OpenShift Log Forwarding API] on the OpenShift blog. After you complete those steps, proceed to the <<integrating-open-liberty-json-logs,Integrating Open Liberty JSON Logs>> and <<setting-up-splunk-dashboard,Setting Up Splunk-Dashboard>> sections of this blog post to complete your setup.
 
 
 == Configuring the Log Forwarding API
 
-. Ensure your cluster logging instance is created and all pods are fully operational. For more information, see the link:https://docs.openshift.com/container-platform/4.6/logging/cluster-logging-deploying.html[Deploying cluster logging] documentation for your OCP version.  The cluster logging instance sample that is provided in this documentation leaves the _storage class_ value for the reader to configure. In test environments, you can omit the storage section and the cluster logging instance uses in-memory storage. However, this configuration is not suitable for production.
```

Comment:
I removed the second sentence because this info is covered in the OpenShift doc, which the user will have to consult anyway in the instance that they are setting up new cluster logging for this guide. I'm assuming that's probably not the case for most users? If you disagree I can restore it.

Reply:
Ah this was an issue brought up during testing by others. The RH doc doesn't tell you _what to do_. So for anybody reading this blog who is _new_ to the whole thing and are trying things they might get stuck.

Reply:
ok I'll restore it

Reply:
restored